libr/flag/Makefile

```diff
@@ -4,4 +4,5 @@ NAME=r_flag
 R2DEPS=r_util
 OBJS=flag.o zones.o tags.o
 
+include $(SHLR)/zip/deps.mk
```

Comment:
why this? on meson r_flag depends only on r_util (which in turn depends on zip)

Reply:
because the build was failing without this dependency